ups/validation_data_jointcal.table

```diff
@@ -0,0 +1,2 @@
+setupRequired(obs_lsstSim)
+setupRequired(obs_cfht)
```

Comment:
Hmm... I'd say they are required to use this particular package (which is an optional dependency on jointcal). jointcal's tests check for the existence of validation_data_jointcal, but not its dependencies. I have to assume if validation_data_jointcal is set up that I can use anything in it.